### PR TITLE
Adding new TAG chairs, adding Emeritus Chairs

### PR DESCRIPTION
**Pull request by amye**

### CODE REVIEW

**Comment by ultrasaurus** [comment]
Security TAG changes look good to me

**Review by TheFoxAtWork — COMMENTED** [review]
Lgtm!

**Review by lumjjb — APPROVED** [review]
LGTM

File: tags/README.md [hunk 1]
```diff
@@ -63,3 +63,7 @@ a pull request with document referencing the roles and charter, updating the lis
 | TAG | Emeritus Chair |
 |---|---|
 | TAG Contributor Strategy | [Gerred Dillon](https://github.com/gerred) |
+| TAG Security | [Sarah Allen](https://github.com/ultrasaurus) |
+| TAG Security |[Jeyappragash Jeyakeerthi](https://github.com/pragashj) |
+| TAG Security |[Dan Shaw](https://github.com/dshaw) |
+| TAG Storage | [Erin Boyd](https://github.com/erinboyd |
```
Comment (lumjjb) [inline tags/README.md:69]:
```suggestion
| TAG Storage | [Erin Boyd](https://github.com/erinboyd) | 
```

**Pull request merged**